main.sh

```diff
@@ -21,4 +21,8 @@ case $key in
 esac
 done
 
-uvicorn app.server:APP --host 0.0.0.0 --port $PORT
+if [[ -z $DEPLOYMENT ]]; then
```

Comment:
I think we should prefer using "flag-like" variables to set these kind of options. So for this, maybe a variable called `USE_GUNICORN`?